src/components/Home.test.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
```

Comment:
- [ ] Kindly add tests that simulate user actions, like clicking buttons and making calculations using the React testing Library.